pkg/3scale/amp/operator/system_reconciler.go

```diff
@@ -127,7 +127,7 @@ func (r *SystemReconciler) Reconcile() (reconcile.Result, error) {
 		reconcilers.DeploymentConfigPodTemplateLabelsMutator,
 		upgrade.SphinxSecretKeyEnvVarMutator,
 	)
-	err = r.ReconcileDeploymentConfig(system.SphinxDeploymentConfig(), sphinxDCmutator)
+	err = r.ReconcileDeploymentConfig(system.SphinxDeploymentConfig(*r.apiManager.Spec.PriorityClassName), sphinxDCmutator)
```

Comment:
same q as above

Reply:
Thank you for comment, please see reply above